php-nginx/whitelist_functions.php

```diff
@@ -14,22 +14,33 @@
     $whitelist = array_filter(array_map('trim', explode(',', $whitelist)));
 }
 
-$suhosinBlacklist = ini_get('suhosin.executor.func.blacklist');
 $disable_functions = ini_get('disable_functions');
 
+// It's tricky, but we still don't have suhosin enabled and we can not fetch
+// the list from our suhosin config. Now it defines the list as a literal.
+// Note: We need to sync this list with our default suhosin config
+// (deb-package-builder/extensions/suhosin/debian/ext-suhosin.ini).
+// TODO: Find a better way
```

Comment:
Yeah, you're sane, but unfortunately no because

* If we enable suhosin in earlier stage, `composer run-script` fails because of suhosin. Suhosin is under simulation mode, but it dies because of `phar` is not allowed. Yeah, seems like it's a suhosin's bug, but now it works like that.

* and whitelist_functions.sh must be executed before `composer run-script` for some cases. An example is in php56_custom test app. The Sympony/Process needs `proc_open` and `proc_close` to be enabled.